### DISCUSSION TITLE
Bug: Cannot read property 'updated' of undefined

### DISCUSSION THREAD

**Discussion by AlexandreDeCarli** [requarks/wiki#3680] [Error / Bug Report]
**Describe the bug**
After I update the wiki from 2.1.113 to 2.2.51 I receive this error: "Bug: Cannot read property 'updated' of undefined"

And at the same time the file conflicts in the interface, although there is no difference between the files.

Although I resolve the conflict, the error message "Cannot read property 'updated' of undefined" keeps showing up.

**To Reproduce**
1. Go to any page from my wiki
2. Click to edit any page
3. Edit any things
4. Click "save"
5. The error "Cannot read property 'updated' of undefined" are shown

**Expected behavior**
I really hope the wiki will save without causing conflicts and errors.

I imagine it will be difficult to reproduce this error, so I need help to help me fix it.

My company has more than 2300 pages created and imagine how much work is giving the team this error.

**Screenshots**

![Video to reproduce this error](https://i.imgur.com/7fMTICr.gif)

**Host Info (please complete the following information):**
OS: Ubuntu 18.04.4 LTS
Wiki.js version: 2.3.77
Database engine: postgres 9.5.21
Node.js: v13.3.0

**Additional context**
PS: Every pages I have to remove the "pt-br" from the URL to be able to edit, otherwise it shows the page to create a new page (see the image gif)

**Comment by ghost** [comment]
I'm having the same problem
